tests/unit_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+# This is for use as a Github CI Pull Request Unit Test.
+echo "ALL_CHANGED FILES is/are ${ALL_CHANGED_FILES}."
+echo "CHANGED_PACKAGES is/are ${CHANGED_PACKAGES}."
+cd /usr/local/lib/crew/packages/
+yes | crew upgrade
+yes | crew install vim
+yes | crew remove vim
+ruby ../tests/commands/const.rb
+ruby ../tests/commands/help.rb
+ruby ../tests/commands/prop.rb
+if [[ -n ${ALL_CHANGED_FILES-} ]]; then
+  # for file in ${ALL_CHANGED_FILES}; do
+    # ruby ../tests/prop_test "$file"
+    # ruby ../tests/buildsystem_test "$file"
+  # done
+  if [[ -n ${CHANGED_PACKAGES-} ]]; then
+    all_compatible_packages=$(crew list -d compatible)
```

Comment:
Do we need to pass `-d here`? doesn't that just disable colors?

Reply:
Yes, because otherwise it spits out color ansi codes... 

It's really useful to be able to do local testing where the TTY will be detected and hence color will automatically be enabled, but that will much up the env variables...